temba/channels/views.py

```diff
@@ -1702,9 +1706,30 @@ class JunebugForm(forms.Form):
                                        required=False)
 
         title = _("Connect Junebug")
-        channel_type = Channel.TYPE_JUNEBUG
         form_class = JunebugForm
-        fields = ('country', 'number', 'url', 'username', 'password')
+        fields = ('channel_type', 'country', 'number', 'url', 'username', 'password')
+
+        def form_valid(self, form):
+            org = self.request.user.get_org()
+
+            if not org:  # pragma: no cover
```

Comment:
This is garbage (that needs to be removed elsewhere as well), you can leave it out.